main.css

```diff
@@ -47,6 +47,21 @@ body {
 	word-spacing: 0.1rem;
 }
 
+::-webkit-scrollbar {
+    width: 12px;
```

Comment:
let's make it narrower
```suggestion
    width: 6px;
```